src/_pytest/logging.py

```diff
@@ -297,6 +297,13 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=None,
         help="Auto-indent multiline messages passed to the logging module. Accepts true|on, false|off or an integer.",
     )
+    group.addoption(
+        "--logger-disable",
```

Comment:
All other flags start with `--log-*`, so perhaps this should be `--log-disable`, or `--disable-log`.

Reply:
Changed to `--log-disable`, as it disables just one logger per argument.